themes/bootstrap3/templates/record/storageretrievalrequest.phtml

```diff
@@ -93,7 +93,7 @@
   </div>
 </form>
 
-<script type="text/javascript">
+<script type="text/javascript" nonce="<?=$this->cspnonce()?>">
```

Comment:
This is another area where it may be better to use the inlinescript helper rather than modify the explicitly coded script tag.

Reply:
changed in follow-up